sql.go

```diff
@@ -25,12 +26,26 @@ func OpenDB() (err error) {
 
 func (z *Zusage) Put() (err error) {
 	log.Println("Put", z)
-	_, _ = db.Exec("DELETE FROM zusagen WHERE nick = $1", z.Nick)
-	_, err = db.Exec("INSERT INTO zusagen (nick, kommt, kommentar) VALUES ($1, $2, $3)", z.Nick, z.Kommt, z.Kommentar)
-	if err != nil {
+
+	old := Zusage{}
+	err = db.QueryRow("SELECT nick, kommt, kommentar FROM zusagen WHERE nick = $1", z.Nick).Scan(&old)
```

Comment:
Exec (which we used before to delete the entry) returns an sql.Result, which has a RowsAffected() method that gives us the number of deleted rows. As far as I know this is supported by postgres. It should be a lot simpler (and cheaper) to just use the result and check if something was deleted.